examples/config/ci/deployer.json

```diff
@@ -0,0 +1,153 @@
+    "AmazonS3PathPrefix": "",
+    "AmazonS3Region": "us-east-1",
+    "AmazonS3Endpoint": "s3.amazonaws.com",
+    "AmazonS3SSL": true,
+    "AmazonS3SignV2": false,
+    "AmazonS3SSE": false
+  },
+  "ExternalAuthProviderSettings": {
+    "Enabled": false,
+    "KeycloakAdminUser": "mmadmin",
+    "KeycloakAdminPassword": "mmpass",
+    "KeycloakRealmFilePath": "",
+    "KeycloakDBDumpURI": "",
+    "GenerateUsersCount": 0
+  },
+  "MattermostDownloadURL": "https://latest.mattermost.com/mattermost-enterprise-linux",
+  "MattermostLicenseFile": "/home/runner/mattermost-load-test",
+  "MattermostConfigPatchFile": "",
+  "AdminEmail": "sysadmin@sample.mattermost.com",
+  "AdminUsername": "sysadmin",
+  "AdminPassword": "Sys@dmin-sample1",
+  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/v1.23.0-rc1/mattermost-load-test-ng-v1.23.0-rc1-linux-amd64.tar.gz",
+  "LogSettings": {
+    "EnableConsole": true,
+    "ConsoleLevel": "INFO",
+    "ConsoleJson": false,
+    "EnableFile": true,
+    "FileLevel": "INFO",
+    "FileJson": true,
+    "FileLocation": "deployer.log",
+    "EnableColor": true
+  },
+  "Report": {
+    "Label": "{instance=~\"app.*\"}",
+    "GraphQueries": [
+      {
+        "Name":  "CPU Utilization",
+        "Query": "avg(rate(mattermost_process_cpu_seconds_total{instance=~\"app.*\"}[1m])* 100)"
+      },
+      {
+        "Name":  "Heap In Use",
+        "Query": "avg(go_memstats_heap_inuse_bytes{instance=~\"app.*:8067\"})"
+      },
+      {
+        "Name":  "Stack In Use",
+        "Query": "avg(go_memstats_stack_inuse_bytes{instance=~\"app.*:8067\"})"
+      },
+      {
+        "Name": "Goroutines In Use",
+        "Query": "sum(go_goroutines{instance=~\"app.*:8067\"})"
+      },
+      {
+        "Name":  "RPS",
+        "Query": "sum(rate(mattermost_http_requests_total{instance=~\"app.*:8067\"}[1m]))"
+      },
+      {
+        "Name":  "Avg Store times",
+        "Query": "sum(rate(mattermost_db_store_time_sum{instance=~\"app.*:8067\"}[1m])) / sum(rate(mattermost_db_store_time_count{instance=~\"app.*:8067\"}[1m]))"
+      },
+      {
+        "Name":  "P99 Store times",
+        "Query": "histogram_quantile(0.99, sum(rate(mattermost_db_store_time_bucket[1m])) by (le))"
+      },
+      {
+        "Name":  "Avg API times",
+        "Query": "sum(rate(mattermost_api_time_sum[1m])) / sum(rate(mattermost_api_time_count[1m]))"
+      },
+      {
+        "Name":  "P99 API times",
+        "Query": "histogram_quantile(0.99, sum(rate(mattermost_api_time_bucket[1m])) by (le))"
+      }
+    ]
+  },
+  "StorageSizes": {
+    "Agent": 10,
+    "Proxy": 10,
+    "App": 15,
+    "Metrics": 50,
+    "Job": 50,
+    "ElasticSearch": 20
+  },
+  "PyroscopeSettings": {
+    "EnableAppProfiling": true,
+    "EnableAgentProfiling": true,
+    "BlockProfileRate": 0
+  },
+  "CustomTags": {
+    "Origin": "ci-automated-loadtests",
+    "PR": "undefined",
+    "SHA": "undefined"
```

Comment:
Are these filled in later? Why the undefined literal vs an empty string?

Reply:
Yes, [they are](https://github.com/mattermost/delivery-platform/pull/222/files#diff-b489de800b182237e408bba9e7d5cd5fba6b55439057d750be59bd94416d62a1R148-R152)! It doesn't really matter what we put here, it could be another literal or an empty string, nothing special about `undefined`, just what I first wrote :)

Reply:
Let's set it to empty. We should actually move to toml where we can add comments.

Reply:
Yup, but we need some more work for toml. I have some changes in a git stash forgotten somewhere.

Reply:
Indeed. I forced myself to spend some time and moved all local load-test configs to toml, and now life is great!